Data/Csv/Encoding.hs

```diff
@@ -144,6 +144,7 @@ decodeByNameWith !opts = decodeWithP (csvWithHeader opts)
 data Quoting
     = QuoteNone        -- ^ No quotes
     | QuoteMinimal     -- ^ Quotes according to RFC 4180
+    | QuoteOld         -- ^ Quotes minimal, or when there are spaces in the field
```

Comment:
I don't see the need for `QuoteOld`, but I guess it's eventually upstreams call.